tests/unit/validation/test_assertions.py

```diff
@@ -28,10 +28,14 @@ def test_validation_exception(errors_to_string_mock: MagicMock):
 def test_assert_valid_input_data(validate_mock: MagicMock):
     validate_mock.return_value = None
     assert_valid_input_data(
-        input_data={"foo": np.array([1])}, calculation_type=CalculationType.state_estimation, symmetric=False  # type: ignore
+        input_data={"foo": np.array([1])},
+        calculation_type=CalculationType.state_estimation,
+        symmetric=False,  # type: ignore
     )
     validate_mock.assert_called_once_with(
-        input_data={"foo": np.array([1])}, calculation_type=CalculationType.state_estimation, symmetric=False  # type: ignore
+        input_data={"foo": np.array([1])},
+        calculation_type=CalculationType.state_estimation,
+        symmetric=False,  # type: ignore
```

Comment:
i think the `type: ignore`s are in the wrong location (i think they should be on `input_data`). 
In addition, `ruff` uses `noqa[CODE]` instead of `type: ignore`. please double-check in #1000 as it's out of scope of this PR.